sas9.4-viya/scripts/mid_install.sh

```diff
@@ -95,21 +92,40 @@ else
     fail_if_error $? "ERROR: Failed to Mount Azure file share"
 fi
 
+#Cloning the sasinstall properties repo
+RETRIES=10
+DELAY=10
+COUNT=1
+while [ $COUNT -lt $RETRIES ]; do
+  git clone https://github.com/corecompete/sasinstalls.git sasinstalls
```

Comment:
all artifacts need to be included in the sample, won't comment on the rest

Reply:
changed the code logic accordingly.